packages/Uploader/src/Uploader.js

```diff
@@ -3,11 +3,17 @@ import PropTypes from "prop-types";
 import useI18n from "@paprika/l10n/lib/useI18n";
 import { containerStyles } from "./Uploader.styles";
 import { getFiles } from "./helpers";
-import ProgressBar from "./components/ProgressBar";
-import types from "./types";
+import FileList from "./components/FileList";
+import DropZone from "./components/DropZone";
+import types from "./statuses";
 import useDragAndDropEvents from "./useDragAndDropEvents";
 import useProcessFiles from "./useProcessFiles";
 
+// TODO: there is a flicker when go over dropzone
+// TODO: why is restarting the 2nd file waiting until the 1st file is complete?
```

Comment:
are these true still or should we remove it?

Reply:
@nahumzs they are true; just wanted to get this PR out there and take a break from this component